### ISSUE TITLE
Photosets not showing up

### ISSUE THREAD

**Issue by TAGrafx** [ronakg/awesome-flickr-gallery-plugin#111]
When I go to my list of photosets on plugin (under flickr), I'm not seeing all of them. All of the settings test perfectly fine.

Albums not showing up
https://www.flickr.com/photos/newyorkgraffiti/albums/72157660171655115
https://www.flickr.com/photos/newyorkgraffiti/albums/72157632727928793

These do not appear in the list of photo sets?

**Comment by TAGrafx** [comment]
Please help it seems like plugin is broken now!

**Comment by TAGrafx** [comment]
http://newyorkgraffiti.com/blog/index.php/old-school/trains/

**Comment by ronakg** [comment]
When I click those links, it doesn't take me to any albums.

Are those private albums? If so, you need to grant permission to plugin to access private albums. By default it only fetches your public albums.

**Comment by TAGrafx** [comment]
yes I would like to keep them private, they have work private for a long time, it just started happening today when I added some more photos to the galleries. I can send you a login to look around in Flickr if that help you too. Let me know where it is secure to send.

**Comment by TAGrafx** [comment]
Keys are OK
User is OK
Select Photosets, they are all broken

**Comment by TAGrafx** [comment]
Updated everything in Flickr App Settings as described in Plugin  Settings. All of the galleries are private, they show up fine in Flickr, but which I click Select Photosets they are all broken.

Also in Plugin settings, I can see the 5 sample images fine and they are correct too.

![image](https://cloud.githubusercontent.com/assets/849729/10656244/e3b63efa-784a-11e5-9a60-798c69f71df3.png)

**Comment by TAGrafx** [comment]
![image](https://cloud.githubusercontent.com/assets/849729/10656262/12fd7854-784b-11e5-9f5b-16ba75c57ad5.png)

**Comment by TAGrafx** [comment]
![image](https://cloud.githubusercontent.com/assets/849729/10656499/50d4207c-784d-11e5-8a2e-40ad8fb8fc11.png)

**Comment by ronakg** [comment]
I'll have to check if something changed on Flickr's API or server side. I'm really busy with my regular job right now, so it might be a while before I can get back to you.

**Comment by TAGrafx** [comment]
About how long, cause this is a pretty big issue for me since I have many accounts counting on me. There is nothing on my side that is different so it must of been the latest update maybe for private albums.

**Comment by ronakg** [comment]
I'm sorry but I'm too busy with my regular job to spend any time on the plugin. And unfortunately I don't see myself being able to work on the plugin for new few weeks, so I don't have a date for you.
